docs/source/resources/02_glossary.rst

```diff
@@ -2,8 +2,8 @@
 
 .. _glossary:
 
-Glossary of Common Resources, Information
-=========================================
+Glossary
```

Comment:
Can be display name on the side bar be a bit more informative?

Reply:
i removed any reference to glossary because it doesn't really function as one. Instead, it will be named `Model Class IDs` for easier reference. The links from the respective model docstrings still links correctly.